DEVELOPING.md

```diff
@@ -9,19 +9,26 @@ Start by [forking](https://github.com/ngageoint/opensphere/fork) the OpenSphere
 * git
 * Java 1.7.0+
 * Node 6+ and npm
+* [Yarn](https://yarnpkg.com/en/) (optional)
 * Python (optional)
 
 Ensure that the executables `git`, `node`, and `java` are in your `PATH`.
 
-[Clone|https://help.github.com/articles/cloning-a-repository/] your repository, and change directory into the clone.
+### Workspace Setup
 
-Install the dependencies by running
+#### Yarn
 
-    $ npm install
+Clone [opensphere-yarn-workspace](https://github.com/ngageoint/opensphere-yarn-workspace), and change directory into `opensphere-yarn-workspace/workspace`.
+
+Clone your fork, then run `yarn install`.
```

Comment:
I was under the impression this needed to be done in the workspace root. If so, perhaps make that more clear?

Reply:
That was my initial assumption, but I later realized yarn will walk up the directory structure, find the workspace root, and run install for the workspace. It doesn't matter where you run it from.